src/emm/emm_cli.py

```diff
@@ -349,5 +230,22 @@ def pull_request(ctx: click.Context, title: Optional[str], body: Optional[str])
     orchestrator.create_pull_request(title, body)
 
 
+# Evergreen subcommands.
+cli.add_command(evg_patch)
+cli.add_command(evg_commit_queue)
+
+# Git subcommands.
+cli.add_command(git_branch_create)
+cli.add_command(git_branch_show)
+cli.add_command(git_branch_switch)
+cli.add_command(git_branch_delete)
+cli.add_command(git_branch_pull)
+cli.add_command(git_branch_update)
+cli.add_command(git_status)
+cli.add_command(git_commit)
+cli.add_command(git_add)
+cli.add_command(git_restore)
```

Comment:
In a similar vein to my comment above, this might be nicer if we had a `git` and `evg` subcommand grouping that were then added to the actual `emm` cli.